tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/populate_route_ids_test.py

```diff
@@ -33,8 +33,10 @@ def setUpClass(cls):
         cls.rule_instance = PopulateRouteIds(cls.project_id, cls.dataset_id,
                                              cls.sandbox_id)
 
+        tables = DRUG_EXPOSURE + VOCABULARY_TABLES
```

Comment:
The integration test failed.  `DRUG_EXPOSURE` is currently a string, not a list of strings.  Try:
```python
[DRUG_EXPOSURE] + VOCABULARY_TABLES
```

Reply:
Yes, I checked and found out. I thought it is also a list. I made the change.